snapshot/pkg/controller/snapshotter/snapshotter.go

```diff
@@ -21,11 +21,11 @@ import (
 	"time"
 
 	"github.com/golang/glog"
+	"github.com/pborman/uuid"
```

Comment:
Is it better to use "github.com/pborman/uuid" than "k8s.io/apimachinery/pkg/util/uuid”?